CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ New features:
 
 Bugfixes:
 
+- Fixed encoding of spaces. Spaces must be replaced by `+` instead of `%20`
```

Comment:
Would you mind adding the PR number and your username to the end of this sentence?

ie. (#24 by @dederer)